transmart-data/test_studies/Makefile

```diff
@@ -1,6 +1,6 @@
 SUBDIRS := $(wildcard */.)
 
-TRANSMART_COPY_VERSION := 17.1-HYVE-5.7
+TRANSMART_COPY_VERSION := $(shell cat ../VERSION)
```

Comment:
should this file be added then? Or is it auto-generated?

Reply:
It is auto-generated here: [transmart-data/build.gradle](https://github.com/thehyve/transmart-core/blob/dev/transmart-data/build.gradle#L152).
This file is always included in distributions (the transmart-data-<version>.zip releases) and is also a dependency for the setupPostgresTest task.